aiohttp/web_exceptions.py

```diff
@@ -83,23 +88,71 @@ class HTTPException(Response, Exception):
     def __init__(self, *,
                  headers: Optional[LooseHeaders]=None,
                  reason: Optional[str]=None,
-                 body: Any=None,
                  text: Optional[str]=None,
                  content_type: Optional[str]=None) -> None:
-        if body is not None:
-            warnings.warn(
-                "body argument is deprecated for http web exceptions",
-                DeprecationWarning)
-        Response.__init__(self, status=self.status_code,
-                          headers=headers, reason=reason,
-                          body=body, text=text, content_type=content_type)
-        Exception.__init__(self, self.reason)
-        if self.body is None and not self.empty_body:
-            self.text = "{}: {}".format(self.status, self.reason)
+        if reason is None:
+            try:
+                reason = RESPONSES[self.status_code][0]
+            except Exception:
+                reason = ''
+
+        if text is None:
+            if not self.empty_body:
+                text = "{}: {}".format(self.status_code, reason)
+        else:
+            if self.empty_body:
+                warnings.warn(
+                    ("text argument is deprecated for HTTP status {} ,"
```

Comment:
No need to have braces around text here

Reply:
Don't we have yet black to handle such styling ussues?

Reply:
Dunno, but it's ugly like this now:(

Reply:
Black reformatting fails on python 3.5.
Will apply it when ready